### ISSUE TITLE
Extended Attributes not preserved when creating Docker containers

### ISSUE THREAD

**Issue by cyphar** [openSUSE/kiwi#555]
The final `tar` archive created by `kiwi` doesn't preserve the xattrs (specifically ones like set-capabilities which are quite important for OS filesystem images).

This is the root cause of openSUSE/docker-containers-build#8.

**Comment by vinyanalista** [comment]
Was that bug affecting ISO images as well? I'm using KIWI [v7.03.38](https://github.com/openSUSE/kiwi/tree/v7.03.38) (the one at [openSUSE:Leap:42.1:Update](https://build.opensuse.org/package/show/openSUSE:Leap:42.1:Update/kiwi)). I can see it's a version released before that bugfix, which was made publicly available at [v7.03.62](https://github.com/openSUSE/kiwi/tree/v7.03.62).

If I build an openSUSE Leap 42.1 LiveDVD and then test it with VirtualBox, I'm not able to ping anything. Invoking the `ping` command, both as common or super user, shows me an error message:

```
# ping 8.8.8.8
ping: error while loading shared libraries: libcap.so.2: cannot stat shared object: Permission denied
```

Googling for it, I came to [a page that links to this issue](https://www.cyphar.com/blog/post/docker-broken-ping).

Also, I found [openSUSE's Bug 728312](https://bugzilla.novell.com/show_bug.cgi?id=728312). The [last comment](https://bugzilla.novell.com/show_bug.cgi?id=728312#c20) at that thread suggests running the command

```
# chkstat -n --set --system /usr/bin/ping /usr/bin/ping6
```

to fix a similar problem. I added that to the `config.sh` script, but it didn't help.

**Comment by cyphar** [comment]
@vinyanalista This specific bug didn't affect ISO images (the issue was that `tar` would strip the extended attributes when we built the image rootfs). Your error message looks like the process you are running as doesn't have permission to execute the `libcap.so.2` shared library.
